src/Libraries/OrchardCore.Commerce.Abstractions/Exceptions/FrontendException.cs

```diff
@@ -1,10 +1,14 @@
 using Microsoft.AspNetCore.Mvc.Localization;
 using System;
-using System.Runtime.Serialization;
+using System.Diagnostics.CodeAnalysis;
 
 namespace OrchardCore.Commerce.Abstractions.Exceptions;
 
 [Serializable]
+[SuppressMessage(
+    "Maintainability",
+    "S3925: ISerializable should be implemented correctly",
+    Justification = "Exception(SerializationInfo info, StreamingContext context) is obsolete and should not be called.")]
```

Comment:
That is actually true. `[Serializable]` can be removed here.

Reply:
Could be turned off but this is the first time I came across a problem rooting from this so I wuoldn't actually turn it off now that I could just remove it.

Reply:
I asked about turning it off because the text in the `Justification` made it sound like you are addressing an outdated analyzer rule by suppressing every occurrence individually. That's not ok even if there is only one case. Since removing `[Serializable]` did the trick we don't have to worry about it.